config/locales/zh.yml

```diff
@@ -1353,6 +1353,8 @@ zh:
     comment:
       wrote: 写到
       send_private_message: "发送私人消息给%{person}"
+      delete: 删除
+      are_you_sure: 你确定要删除吗？
```

Comment:
All language changes and additions (expect English) should be done with WebTranslateIt. Otherwise, the WTI tool may override the local changes.

So what you should do is to translate these two keys from WTI, after the feature has been merged to master.